libs/components/src/lib/Accordion/index.module.scss

```diff
@@ -0,0 +1,88 @@
+@import "../../../../../libs/theme";
+
+.container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  &__item {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    border-bottom: 3px solid $grey-200;
+    transition: all 0.3s ease-in-out;
+
+    &:last-child {
+      border: none;
+    }
+
+    &__header {
+      width: 100%;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      justify-content: space-between;
+      padding: 24px 0 18px 0;
+      cursor: pointer;
+
+      &__title {
+        font-size: 20px;
+        color: $dark;
+        font-weight: 600;
+
+        &__open {
+          color: $blue !important;
+          font-weight: 700;
+        }
+      }
+
+      &__icon {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        transition: all 0.3s ease-in-out;
+
+        &__open {
+          transform: rotate(135deg);
+        }
+      }
+    }
+
+    &__title {
+      width: 100%;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      justify-content: center;
+      cursor: pointer;
+    }
+
+    &__children {
+      width: 100%;
+      font-size: 16px;
+      display: flex;
+      align-items: center;
+      max-height: 0;
+      opacity: 0;
+      transition: all 0.3s ease-in-out;
+
+      overflow: hidden;
+      color: $dark;
+      font-family: Montserrat;
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 400;
+
+      &__open {
+        max-height: 1000px; /* or any other large value */
```

Comment:
remove comment, 1000px ??

Reply:
removed comment and reduced max-height from 1000 to 800px